src/services/types.ts

```diff
@@ -121,6 +121,7 @@ namespace ts {
 
     export interface HostCancellationToken {
         isCancellationRequested(): boolean;
+        throttleWaitMilliseconds?: number;

```

Comment:
this should be an implementation detail of `ThrottledCancellationToken` and should not be exposed here